Sources/Services/AWSRoute53/models/Models.swift

```diff
@@ -96,31 +96,6 @@ extension Route53ClientTypes {
     }
 }
 
-public struct ActivateKeySigningKeyInputStripHostedZoneMiddleware: ClientRuntime.Middleware {
```

Comment:
14 generated middlewares are removed from Models, because the reusable one is used instead.